module/VuFind/src/VuFind/Config/Feature/EmailSettingsTrait.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * Trait providing email settings
+ *
+ * PHP version 8
+ *
+ * Copyright (C) The National Library of Finland 2024.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Config
+ * @author   Ere Maijala <ere.maijala@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Site
+ */
+
+namespace VuFind\Config\Feature;
+
+use Laminas\Config\Config;
+
+/**
+ * Trait providing email settings
+ *
+ * @category VuFind
+ * @package  Config
+ * @author   Ere Maijala <ere.maijala@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Site
+ */
+trait EmailSettingsTrait
```

Comment:
I wonder if this should be EmailSenderSettingsTrait (or something else more specific) since other user-oriented email settings are handled by \VuFind\Config\AccountCapabilities. Or maybe we should just add a comment noting that for the benefit of anyone who ends up here looking for that code.

Reply:
I added a comment. I don't want to make this too specific to make it possible to add more (non-sender) settings later on if needed.